lua/typescript-tools/autocommands/jsx_close_tag.lua

```diff
@@ -0,0 +1,83 @@
+local api = vim.api
+
+local common = require "typescript-tools.autocommands.common"
+local plugin_config = require "typescript-tools.config"
+local plugin_api = require "typescript-tools.api"
+
+local M = {}
+
+--- @generic T
+--- @param list T[]
+--- @param value T
+--- @return boolean
+local function list_contains(list, value)
+  if vim.list_contains ~= nil then
+    return vim.list_contains(list, value)
+  end
+  for _, v in ipairs(list) do
+    if value == v then
+      return true
+    end
+  end
+
+  return false
+end
+
+function M.setup_jsx_close_tag_autocmds()
+  local augroup = vim.api.nvim_create_augroup("TypescriptToolsJSXCloseTagGroup", { clear = true })
+
+  common.create_lsp_attach_augcmd(function()
+    local changing = false
+    local request_id = nil
+    local prev_line = nil
+    local prev_line_text = ""
+
+    api.nvim_create_autocmd({ "TextChangedI" }, {
+      pattern = { "*" },
```

Comment:
I think this should have pattern to match js/ts files, you can simply import and use [this one](https://github.com/pmizio/typescript-tools.nvim/blob/master/lua/typescript-tools/autocommands/common.lua#L6).
Or map filetypes from config to file extensions and put them here.